_includes/pbc/all/install-features/202307.0/install-glue-api/install-the-merchant-relationship-product-lists-glue-api.md

```diff
@@ -0,0 +1,124 @@
+
+
+## Install Merchant Relationship Product Lists Feature API
```

Comment:
module is mapped to another feature

Reply:
The title was already adjusted to `Install the Catalog + Merchant Product Restrictions feature`.